lib/ChGovUk/Controllers/Admin/User/Filings.pm

```diff
@@ -35,8 +35,9 @@ sub list {
                     $doc->{closed_at_date} = CH::Util::DateHelper->isodate_as_string($doc->{closed_at});
                     $doc->{closed_at_time} = CH::Util::DateHelper->isotime_as_string($doc->{closed_at})->strftime("%l:%M%P");
                 }
-                if ($doc->{status} eq "open" && $doc->{resume_journey_uri}) {
-                   $self->_build_resume_link($doc, $doc->{resume_journey_uri});
+                if (($doc->{status} eq "open" && $doc->{resume_journey_uri})
```

Comment:
This check can be simplified to 'if (status = open or closed pending payment) & resume_journey_uri)' - no need to check the existent of resume journey uri twice